labs/dxp/7.x/installation-and-upgrades/02-setting-up-liferay-dxp/installing-and-managing-apps/installing-apps-via-the-ui.md

```diff
@@ -0,0 +1,31 @@
+# Installing Apps via the UI
+
+Installing apps via the UI leverages auto deployment; the app will install (from the LPKG, module JAR, or plugin WAR) while the DXP server is running. Auto deployment copies the app to the appropriate subfolder in `[Liferay Home]/osgi`, converts it to an OSGi bundle (LPKGs and WARs need converting), and deploys the bundle to the OSGi container.
```

Comment:
I think it's good to highlight the convenience of installing via the UI up front, and leave the details for the followup sentences.

for example,

You can use DXP's app management UIs to install apps to local and remote DXP servers while they're running. Behind the scenes, DXP copies the app file (Marketplace LPKG, module JAR, or plugin WAR) to the appropriate subfolder in `[Liferay Home]/osgi`, converts it to an OSGi bundle (LPKGs and WARs need converting), and deploys the bundle to the OSGi container. This process is managed by DXP's auto deployment process. 

Apps that aren't designed for auto deployment, however, can cause class loading issues and memory leaks.

Reply:
@jhinkey I can get behind the general change here. One part I am unsure about though is just this sentence: 

> Apps that aren't designed for auto deployment, however, can cause class loading issues and memory leaks.

I don't think it's good to say this if we don't explain how someone should know whether or not an app is designed for auto deployment. Is there a way we can make this distinction for someone? (I'm not sure myself how one would tell whether or not an app is designed for auto deployment)

Reply:
I'll ask the core infra team about determining whether an app is designed for auto-deploy, as I don't know how to determine it.